content/synthetics/browser_check.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Browser Check
+kind: documentation
+beta: true
+description: Simulate and monitor user journeys from specific locations.
+further_reading:
+- link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
+  tag: "Blog"
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/uptime_check"
+  tag: "Documentation"
+  text: "Configure an Uptime Check"
+---
+
+<div class="alert alert-warning">Synthetics is in beta for the Datadog US Site. Request access via the <a href="https://app.datadoghq.com/synthetics/beta">Datadog Synthetics request form</a>.</div>
+
+## Overview
+
+Browser Checks are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These checks verify both that your applications are up and responding to requests, and that any conditions defined in your scenarios are met.
+
+## Configuration
+
+### Test details
+
+Define the configuration of your Browser Check.
+
+{{< img src="synthetics/browser_check/browser_check_configuration.png" alt="Browser Check make request" responsive="true" style="width:80%;">}}
+
+1. Give your Browser Check a name. 
+2. Enter the Start URL. This is the URL from which your Browser Check starts the scenario.
```

Comment:
Are we not talking about tags on purpose? It might be worth adding a blurb on the ability to select tags that will allow filtering on the main Synthetics page afterwards?

Reply:
Good idea!